packages/search/tests/unit/services/navi-report-search-provider-test.js

```diff
@@ -0,0 +1,49 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+
+module('Unit | Service | navi-report-search-provider', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function() {
+    // Load metadata needed for request fragment
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    this.service = this.owner.lookup('service:navi-search/navi-report-search-provider');
+  });
+
+  test('it exists', function(assert) {
+    const service = this.owner.lookup('service:navi-search/navi-report-search-provider');
+    assert.ok(service);
+  });
+
+  test('search by user', async function(assert) {
```

Comment:
We should have tests to confirm that the query string is being constructed correctly. These tests are okay but they rely on the mirage endpoint so it's not truly a unit test.